apps/meteor/client/components/FilterByText.tsx

```diff
@@ -46,7 +46,7 @@ const FilterByText = ({
 	}, []);
 
 	return (
-		<Box mb='x16' is='form' onSubmit={handleFormSubmit} display='flex' flexDirection={shouldFiltersStack ? 'column' : 'row'} {...props}>
+		<Box mb='x16' is='form' onSubmit={handleFormSubmit} display='flex' flexDirection={shouldFiltersStack ? 'column' : 'row'}>
```

Comment:
Have you checked if the spread props is not necessary in every usage case?

Reply:
Considering the properties this component accepts, there's nothing relevant to spread.